app/graphql/mutations/create_completed_submission_report.rb

```diff
@@ -24,14 +24,19 @@ def save_report
           SubmissionReport.find_by(submission_id: @params[:submission_id])
         if report.present?
           report.update!(
+            status: 'completed',
             description: @params[:description],
-            status: @params[:status]
+            conclusion: @params[:conclusion],
+            started_at: report.started_at || Time.zone.now,
+            completed_at: Time.zone.now
           )
         else
           SubmissionReport.create!(
-            submission_id: @params[:submission_id],
+            status: 'completed',
             description: @params[:description],
-            status: @params[:status]
+            conclusion: @params[:conclusion],
+            started_at: Time.zone.now,
+            completed_at: Time.zone.now
```

Comment:
Multiple `Time.zone.now` calls could result in subtly different time values. Cache it in a `time_now` method.

```ruby
def time_now
  @time_now ||= Time.zone.now
end
```